data-guides/appendix/glossary.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
I didn't make any changes in `/data-guides/` folder, other than changing its name. What are these or where they stand, I do not know exactly.

Should I move this somewhere?